src/modules/temperature.cpp

```diff
@@ -34,6 +39,15 @@ auto waybar::modules::Temperature::update() -> void {
   } else {
     label_.get_style_context()->remove_class("critical");
   }
+
+  if(format.empty())
+  {
+    event_box_.hide();
+    return;
+  } else {
+    event_box_.show();
+  }
```

Comment:
Just lint and we'll be good :)

Reply:
Thanks. Please check the new one commit :)